src/components/ShareLink.js

```diff
@@ -41,7 +41,8 @@ export default function ShareLink(props) {
   };
 
   const handleOpenShare = async () => {
-    const encoded = deflateSync(JSON.stringify(props.config) + '\n' + props.shareText).toString('base64');
+    const itsyBitsyConfig = { c: props.config?.canonical, v: props.config?.version, d: props.config?.dependencies };
```

Comment:
One question/idea I had was if we could make this even itsy bitsier. If there is no canonical or version or dependencies to share, does it make sense to not include any config + `\n` in the string that we compress?

Reply:
Yeah I think that is fair, why not save the space. And it should work fine since we've already supported that case for legacy links. I'll make the change.

Reply:
Should be good in https://github.com/FSHSchool/FSHOnline/pull/65/commits/f1021072b93b47287cae09d785c7739f17d096ef.